src/lib/openapi/spec/personal-dashboard-schema.ts

```diff
@@ -8,6 +8,35 @@ export const personalDashboardSchema = {
     additionalProperties: false,
     required: ['projects', 'flags'],
     properties: {
+        projectOwners: {
+            type: 'array',
+            description:
+                'Users with the project owner role in Unleash. The list will not include owners of private projects that the user is not a member of, unless the owner is also an owner of a project that the is visible to the user.',
```

Comment:
"unless the owner is also an owner of a project that the is visible to the user" isn't this repeating the same thing as "The list will not include owners of private projects that the user is not a member of"

Reply:
Yes; yes it is. I just couldn't think of a better way to word it in the moment. Thanks for the suggestion?